chain/ethereum.ts

```diff
@@ -360,6 +360,44 @@ export namespace ethereum {
     ) {}
   }
 
+  /**
+   * An Ethereum transaction receipt.
+   */
+  export class TransactionReceipt {
+    constructor(
+      public transactionHash: Bytes,
+      public transactionIndex: BigInt,
+      public blockHash: Bytes,
+      public blockNumber: BigInt,
+      public cumulativeGasUsed: BigInt,
+      public gasUsed: BigInt,
+      public contractAddress: Address,
+      public logs: Array<Log>,
+      public status: BigInt,
+      public root: Bytes,
+      public logsBloom: Bytes,
+    ) {}
+  }
+
+  /**
+   * An Ethereum event log.
+   */
+  export class Log {
+    constructor(
+      public address: Address,
+      public topics: Array<Bytes>,
+      public data: Bytes,
+      public blockHash: Bytes,
+      public blockNumber: Bytes,
+      public transactionHash: Bytes,
+      public transactionIndex: BigInt,
+      public logIndex: BigInt,
+      public transactionLogIndex: BigInt,
+      public logType: string,
+      public removed: boolean,
```

Comment:
This is incorrect, it should be `Wrapped<bool>`

Reply:
Fixed in e286328